lib/index.js

```diff
@@ -151,25 +151,29 @@ PostHTML.prototype.process = function (tree, options) {
 
   // sync mode
   if (options.sync === true) {
-    this.plugins.forEach(function (plugin, index) {
+    if (this.plugins.length > 0) {
```

Comment:
@fansenze plugins cannot be empty since https://github.com/posthtml/posthtml/blob/83d66e505bf302d0ad7f34ecc527c371821169fa/lib/index.js#L27

Reply:
`[]` not empty?
i try to use process() with `sync: true`, then i can't call the `walk/match` in `tree`
it means ok?

Reply:
`[] not empty?`

Of course it’s not empty but there is no need to check

Reply:
> i try to use process() with sync: true, then i can't call the walk/match in tree
it means ok?

this is not due to the presence of plugins in `this.plugins`